example/src/test/scala/monocle/function/WrappedExample.scala

```diff
@@ -0,0 +1,26 @@
+package monocle.function
+
+import monocle.MonocleSuite
+import monocle.function.all.{ala, unwrapped, wrapped}
+
+import scalaz.{@@, Tags}
+import scalaz.std.anyVal._
+import scalaz.std.list._
+import scalaz.std.function._
+import scalaz.syntax.foldable._
+
+class WrappedExample extends MonocleSuite {
+
+  test("wrapped is an Iso") {
+    (Tags.Max(100) applyIso wrapped[Int @@ Tags.Max, Int] get) shouldEqual 100
```

Comment:
are the type annotations necessary? if yes, it really limits the usefulness of `wrapped`